examples/analytics.amp.html

```diff
@@ -237,14 +237,21 @@
 </amp-analytics>
 <!-- End comScore example -->
 
-<!-- INFOnline example -->
+<!--
+  INFOnline example
+
+  Important: url needs to point to a copy of
+  amp-analytics-infonline.html on a different subdomain
+  than your AMP files.
+-->
 <amp-analytics type="infonline" id="infonline">
 <script type="application/json">
 {
   "vars": {
     "st":  "angebotskennung",
     "co":  "comment",
-    "cp":  "code"
+    "cp":  "code",
+    "url":  "https://3p.ampproject.net/custom/amp-analytics-infonline.html"
```

Comment:
This can be specified as a `requests` entry and then used as a variable that is not encoded.